testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/KeycloakRealmImportOperatorProvisionerTest.java

```diff
@@ -0,0 +1,373 @@
+import org.keycloak.k8s.v2alpha1.keycloakspec.Http;
+import org.keycloak.k8s.v2alpha1.keycloakspec.Ingress;
+import org.keycloak.k8s.v2alpha1.keycloakspec.db.PasswordSecret;
+import org.keycloak.k8s.v2alpha1.keycloakspec.db.UsernameSecret;
+import org.slf4j.event.Level;
+
+import cz.xtf.core.openshift.OpenShiftWaiters;
+import cz.xtf.core.openshift.OpenShifts;
+import cz.xtf.core.waiting.SimpleWaiter;
+import cz.xtf.junit5.annotations.CleanBeforeAll;
+import io.fabric8.kubernetes.api.model.DeletionPropagation;
+import io.fabric8.kubernetes.api.model.KubernetesResourceList;
+import io.fabric8.kubernetes.client.dsl.NonNamespaceOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Test the Keycloak Operator provisioning model and APIs
+ *
+ * See <br>
+ *     - https://github.com/keycloak/keycloak-operator/tree/master/deploy/examples
+ * <p>
+ * Not all examples are actually run, but these tests are more about the framework functionality verification than
+ * about the Keycloak operator testing.
+ */
+@Slf4j
+@CleanBeforeAll
+//@Disabled("WIP - Disabled until global-test.properties is configured with the required property")
+public class KeycloakRealmImportOperatorProvisionerTest {
+	private static KeycloakRealmImportOperatorProvisioner KEYCLOAK_OPERATOR_PROVISIONER;
+
+	private static final String POSTGRESQL_NAME = "postgresql";
+	private static final String POSTGRESQL_DATABASE = "keycloak";
+	private static final String POSTGRESQL_PASSWORD = "pippobaudo1234";
+	private static final String POSTGRESQL_USER = "user09M";
+
+	private static final PostgreSQLImageOpenShiftApplication pgSQLApplication = new PostgreSQLImageOpenShiftApplication() {
+		@Override
+		public String getName() {
+			return POSTGRESQL_NAME;
+		}
+
+		@Override
+		public String getUser() {
+			return POSTGRESQL_USER;
+		}
+
+		@Override
+		public String getPassword() {
+			return POSTGRESQL_PASSWORD;
+		}
+
+		@Override
+		public String getDbName() {
+			return POSTGRESQL_DATABASE;
+		}
+	};
+	private static final PostgreSQLImageOpenShiftProvisioner POSTGRESQL_IMAGE_PROVISIONER = new PostgreSQLImageOpenShiftProvisioner(
+			pgSQLApplication);
+
+	private static KeycloakRealmImportOperatorProvisioner initializeOperatorProvisioner(final Keycloak keycloak,
+			final String appName) {
+		KeycloakRealmImportOperatorProvisioner operatorProvisioner = new KeycloakRealmImportOperatorProvisioner(
+				new KeycloakRealmImportOperatorApplication() {
+
+					@Override
+					public Keycloak getKeycloak() {
+						return keycloak;
+					}
+
+					@Override
+					public String getName() {
+						return appName;
+					}
+				});
+		return operatorProvisioner;
+	}
+
+	private String name;
+	private String realmName;
+
+	private static final Map<String, String> matchLabels = new HashMap<>();
+
+	@BeforeAll
+	public static void createOperatorGroup() throws IOException {
+		matchLabels.put("app", "sso");
```

Comment:
Here is where `configure()` should be called, see e.g.: https://github.com/Intersmash/intersmash/blob/main/testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/InfinispanOperatorProvisionerTest.java#L99

Reply:
Since the dummy configuration was removed, also the `configure()` method was moved after the provisioner is actually initialized (e.g. inside test `public void exampleSso()`)

Reply:
This can be resolved since the provisioner life cycle will be managed by the test methods. Resolving.